js/pages/data-sources/data-sources.less

```diff
@@ -17,16 +21,18 @@
     }
   }
 
-  &.dataSourceReport {
-    width: 96%;
-    margin-left: auto;
-    margin-right: auto;
-    margin-top: 0px;
-  }
+	& .dataSourceReportHeading {
```

Comment:
I think since this block is being used on every report, it sould be a separate component with its own `.less` file, controller and a template

Reply:
reverted to shared heading component
https://github.com/OHDSI/Atlas/pull/1165/commits/d96ab3957d664838b3b70f87a65f095b9278e827